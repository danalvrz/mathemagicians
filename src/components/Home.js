import React from 'react';
import Header from './Header';

const Home = () => (
  <div>
    <Header />
    <section className="homeContainer">
      <h2 className="homeTitle">
        Welcome to the (number) jungle!
      </h2>
      <div className="homeTextContainer">
        <p className="homeText" style={{ color: 'aliceblue' }}>
          Lorem ipsum dolor sit amet consectetur adipisicing elit. Dicta
          doloremque dolores dolorum alias dignissimos fuga praesentium nemo
          quaerat molestiae expedita ut illo vel eveniet placeat, totam omnis sit
          maiores officia nam ea numquam. Possimus doloribus culpa, sit itaque
          quidem, modi minus voluptatem totam vero vitae voluptas a natus
          explicabo rem!
        </p>
        <p className="homeText" style={{ color: 'aliceblue' }}>
          Lorem ipsum dolor sit amet consectetur adipisicing elit. Dicta
          doloremque dolores dolorum alias dignissimos fuga praesentium nemo
          quaerat molestiae expedita ut illo vel eveniet placeat, totam omnis sit
          maiores officia nam ea numquam. Possimus doloribus culpa, sit itaque
          quidem, modi minus voluptatem totam vero vitae voluptas a natus
          explicabo rem!
        </p>
      </div>
    </section>
  </div>
);

export default Home;
