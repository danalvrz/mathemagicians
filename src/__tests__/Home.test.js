import React from 'react';
import renderer from 'react-test-renderer';
// eslint-disable-next-line no-unused-vars
import { render, fireEvent } from '@testing-library/react';
import { HashRouter as Router } from 'react-router-dom';
import Home from '../components/Home';

test('renders correctly', () => {
  const tree = renderer.create(
    <Router>
      <Home />
    </Router>,
  ).toJSON();
  expect(tree).toMatchSnapshot();
});
