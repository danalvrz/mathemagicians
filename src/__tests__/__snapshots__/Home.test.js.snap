// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`HomeTesting renders correctly 1`] = `
<div>
  <header
    className="menuContainer"
  >
    <h1>
      Mathemagicians
    </h1>
    <nav
      className="menuList"
    >
      <a
        href="#/"
        onClick={[Function]}
      >
        Home
      </a>
      |
      <a
        href="#/calculator"
        onClick={[Function]}
      >
        Cowculator
      </a>
      |
      <a
        href="#/quote"
        onClick={[Function]}
      >
        Quote
      </a>
    </nav>
  </header>
  <section
    className="homeContainer"
  >
    <h2
      className="homeTitle"
    >
      Welcome to the (number) jungle!
    </h2>
    <div
      className="homeTextContainer"
    >
      <p
        className="homeText"
        style={
          Object {
            "color": "aliceblue",
          }
        }
      >
        Lorem ipsum dolor sit amet consectetur adipisicing elit. Dicta doloremque dolores dolorum alias dignissimos fuga praesentium nemo quaerat molestiae expedita ut illo vel eveniet placeat, totam omnis sit maiores officia nam ea numquam. Possimus doloribus culpa, sit itaque quidem, modi minus voluptatem totam vero vitae voluptas a natus explicabo rem!
      </p>
      <p
        className="homeText"
        style={
          Object {
            "color": "aliceblue",
          }
        }
      >
        Lorem ipsum dolor sit amet consectetur adipisicing elit. Dicta doloremque dolores dolorum alias dignissimos fuga praesentium nemo quaerat molestiae expedita ut illo vel eveniet placeat, totam omnis sit maiores officia nam ea numquam. Possimus doloribus culpa, sit itaque quidem, modi minus voluptatem totam vero vitae voluptas a natus explicabo rem!
      </p>
    </div>
  </section>
</div>
`;
